yarn-project/p2p/src/services/libp2p/libp2p_service.ts

```diff
@@ -1038,7 +1065,10 @@ export class LibP2PService<T extends P2PClientType = P2PClientType.Full> extends
         severity = await this.handleDoubleSpendFailure(tx, txBlockNumber);
       }
 
-      this.peerManager.penalizePeer(peerId, severity);
+      // In fisherman mode, we validate but don't penalize peers
+      if (!this.config.fishermanMode) {
+        this.peerManager.penalizePeer(peerId, severity);
+      }
```

Comment:
Why not? I think we'll want our fisherman node to connect to healthy nodes, so it doesn't get an incorrect view of the p2p network if it gets stuck connected to faulty nodes that other nodes are dropping.